firstvoices/backend/views/join_request_views.py

```diff
@@ -162,7 +163,14 @@ def reject(self, request, site_slug=None, pk=None):
             join_request, JoinRequestStatus.REJECTED, request.user
         )
 
-        # notify user here, see FW-5077
+        subject = f"Thank you for your interest in joining the FirstVoices site {join_request.site.title}."
+        message = (
+            f"Thank you for your interest in joining the FirstVoices site {join_request.site.title}.\n\n"
+            f"At this time the site is not accepting new members and your request to join the site has not been "
+            f"approved.\n\n"
+            f"If you think this may be an error please contact hello@firstvoices.com.\n\n"
```

Comment:
Does text of these messages have the Clarissa stamp of approval?

Reply:
Had a chat with Clarissa and made some changes to the messages. Will merge this PR after a final review from Bridget.